src/app/components/audio-recordings/pages/download/download.component.ts

```diff
@@ -0,0 +1,389 @@
+          </li>
+          <li>
+            Click on the containing folder, and hit the "Copy path" button in
+            the Clipboard section of the context menu (top menu, Home tab). The
+            location path you have copied should look similar to
+            <code>C:/Users/username/Downloads</code>
+          </li>
+          <li>Open a new terminal</li>
+          <li>Run <code>pwsh</code></li>
+          <li>
+            Run the following command, replace the text in-between the quotes
+            with the path you just copied:
+            <code>cd "FOLDER_LOCATION_HERE"</code>
+          </li>
+        </ol>
+
+        <h4>2.2. Linux</h4>
+
+        <ol>
+          <li>Open the location of the script in your distros file explorer</li>
+          <li>
+            Most file explorers in linux have the ability to open a terminal at
+            the current location of the file explorer, use this option if
+            available
+          </li>
+          <li>
+            Otherwise, copy the containing folder location for the file. It
+            should look something similar to
+            <code>/home/username/Downloads</code>
+          </li>
+          <li>Open a new terminal</li>
+          <li>
+            Run the following command, replace the text in-between the quotes
+            with the path you just copied:
+            <code>cd "FOLDER_LOCATION_HERE"</code>
+          </li>
+          <li>Run <code>pwsh</code></li>
+        </ol>
+
+        <h4>2.3. MacOS</h4>
+
+        <p>
+          Attempt to follow the Linux instructions if needed, otherwise go to
+          the <a [strongRoute]="contactUs.route">{{ contactUs.label }}</a> page
+          for more help.
+        </p>
+
+        <h3>3. Get Authentication Token</h3>
+
+        <p>
+          Every time you want to run this script, you will need to update the
+          authentication token (auth token). This is what will identify you to
+          our servers, and you can think of it as being similar to your username
+          and password.
+          <strong class="text-danger">
+            Treat this code like your password, and do not ever share it with
+            anyone!
+          </strong>
+          You can find the auth token in your
+          <a [strongRoute]="profile.route">{{ profile.label }}</a> page, under
+          the "Authentication Token" section. Copy the auth token, and keep it
+          for the next step<br />
+          You should perform this step whenever you wish to run the script as
+          the auth token can expire if left for a long enough period of time.
+        </p>
+
+        <h3>4. Run the script</h3>
+
+        <p>
+          Finally we can run the script. Read through the documentation of the
+          script for more specific advice/use cases. To run the command, take
+          the terminal you setup in the previous steps, and run the following
+          command:
+          <code>./download_audio_files.ps1</code>
```

Comment:
~~this invocation is wrong - an auth token parameter is required~~

oh i see: you still missed out on telling them to paste the token